pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -509,8 +512,8 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1beta1.PipelineRun, get
 	// Ensure that the array reference is not out of bound
 	if err := resources.ValidateParamArrayIndex(ctx, pipelineSpec, pr); err != nil {
 		// This Run has failed, so we need to mark it as failed and stop reconciling it
-		pr.Status.MarkFailed(ReasonObjectParameterMissKeys,
-			"PipelineRun %s/%s parameters is missing object keys required by Pipeline %s/%s's parameters: %s",
+		pr.Status.MarkFailed(ReasonParamArrayIndexingInvalid,
+			"PipelineRun %s/%s array indexing params fail validation by Pipeline %s/%s's parameters: %s",
```

Comment:
```suggestion
			"Failed to validate PipelineRun %s/%s: failed to validate Pipeline %s/%s's parameters: %s",
```

Reply:
I think we cannot use "Failed" as the first word, it fails the linter of duplicate workds. 😂